pkg/agent/run.go

```diff
@@ -43,7 +45,11 @@ import (
 	utilpointer "k8s.io/utils/pointer"
 )
 
+const healthControllerChannelLength = 10
+
 func run(ctx context.Context, cfg cmds.Agent, proxy proxy.Proxy) error {
+	var wg sync.WaitGroup
```

Comment:
We're still not doing anything with the waitgroup; if we're not waiting on it (which I don't think we want to do) then lets remove it from the flannel.Run and netpol.Run signatures and allow them to continue to make their own as needed?

Reply:
Alright, I moved the WaitGroup only to netpol.go, since it's impossible to run kube-router without passing `wg`. Then I'm not doing anything with it.